study.md

```diff
@@ -20,21 +20,22 @@ material. Instead, digest what you've read and repeat it in your own voice.
 In your own words, describe the event loop and how it works.
 
 ```md
-<!-- your answer here -->
+<!-- The event loop is when a current function runs in task queue and event loop checks if stack is empty then allows task in through queue to stack, where the job gets done.  -->
 ```
 
 ## Synchronicity and Asynchronicity
 
 In your own words, describe the difference between sync and async.
 
 ```md
-<!-- your answer here -->
+<!-- Sync is when callback runs in order and async does not run in a specific order. -->
 ```
```

Comment:
Sync actually runs in a very specific order, but it can run multiple parts of our code at once, as opposed to only one after the other.